consoles/video_stream.pm

```diff
@@ -0,0 +1,340 @@
+# Copyright 2021 Marek Marczykowski-Górecki
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package consoles::video_stream;
+
+use Mojo::Base -strict, -signatures;
+
+use base 'consoles::network_console';
```

Comment:
I suggest to introduce a new base class `consoles::video_base` which you can base `consoles::video_stream` and `consoles::vnc_base` on. Then you can prevent any additional duplication like the methods "get_last_mouse_set", "screen", "type_string", the button call Martchus mentioned, etc.